lib/pbench/server/indexing_tarballs.py

```diff
@@ -459,18 +458,17 @@ def sighup_handler(*args):
                                         # accomplish this by overwriting each
                                         # duplicate index key separately.
                                         try:
-                                            meta = Metadata.get(
+                                            map: JSON = Metadata.getvalue(
```

Comment:
That caught Nick, too. It's just an assignment type hint, because I thought it would be helpful to clearly document exactly what we're getting from the metadata key. Since it seems to be causing more confusion than clarity, one might conclude that I was over-zealous, and perhaps I ought to just drop it.

Reply:
Ah, so the name `map` has a type-hint, `JSON`, when assigned the value from `Metadata.getvalue(...)`.

The formatting is tough.  Perhaps something like `map:JSON = ...`?

Bummer that it is removed, a comment would have been sufficient, but okay that it is removed.